src/chunk/chunk_unknown.rs

```diff
@@ -0,0 +1,50 @@
+use std::any::Any;
+use std::fmt::{Debug, Display, Formatter};
+use bytes::{Bytes, BytesMut};
+use crate::chunk::Chunk;
+use crate::chunk::chunk_header::{CHUNK_HEADER_SIZE, ChunkHeader};
+
+#[derive(Clone, Debug)]
+pub struct ChunkUnknown {
+    hdr: ChunkHeader,
+    value: Bytes,
+}
+
+impl Display for ChunkUnknown {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(f, "ChunkUnknown( {} {:?} )", self.header(), self.value)
```

Comment:
do we really need to clone header every time we display it?

Reply:
No not really. But I didn't think the display impl would be that concerned with performance.